DumpCementeryDB.sql

```diff
@@ -19,106 +19,106 @@
 DB Structure Goes Here...
 */
 
-/* ####### Create Sector ####### */
+/* ####### Create Sectors ####### */
 
-DROP TABLE IF EXISTS `sektor`;
-CREATE TABLE `sektor` (
-  `id` int(11) NOT NULL AUTO_INCREMENT
+DROP TABLE IF EXISTS `Sectors`;
+CREATE TABLE `Sectors` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  'number' varchar(15)
```

Comment:
Do we need column number? I think id is enough for sector and it can be sector number

Reply:
As we agreed twice, it was roman number od a row in cementary, which has graves, which are numbered